### PR TITLE
(feat)chart: Add chart for OpenEBS pool failure

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: udit <uditgaurav@gmail.com>

Added the following experiments for OpenEBS charts:
- [x] openebs-pool-failure 

Here are the logs :
[logs-opf.txt](https://github.com/litmuschaos/community-charts/files/3642326/logs-opf.txt)

### CODE REVIEW

File: charts/openebs/network/experiments/openebs-pool-faliure.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+spec:
+  definition:
+    args:
+    - -c
+    - ansible-playbook ./experiments/chaos/openebs_pool_failure/test.yml -i /etc/ansible/hosts
+      -vv; exit 0
+    command:
+    - /bin/bash
+    - name: ANSIBLE_STDOUT_CALLBACK
+      value: default
```
Comment (chandankumar4) [inline charts/openebs/network/experiments/openebs-pool-faliure.yml:20]: Put one blank line space between each value pair, And write a comment for the optional value!!

File: charts/openebs/network/experiments/openebs-pool-faliure.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - name: APP_PVC
+      value: percona-mysql-claim
+    - name: LIVENESS_APP_LABEL
+      value: ""
+    - name: LIVENESS_APP_NAMESPACE
+      value: ""
+    - name: DATA_PERSISTENCE
+      value: ""
+    - name: CHAOS_TYPE
+      value: "pool-kill"
```
Comment (chandankumar4) [inline charts/openebs/network/experiments/openebs-pool-faliure.yml:36]: What's the use of this value?

**Comment by chandankumar4** [comment]
@uditgaurav What's the reason for closing this PR, Is there any new PR raised with respect to this?

**Comment by uditgaurav** [comment]
yes @chandankumar4 a new PR has been raised for this task.

**Pull request closed without merging**